src/io.c

```diff
@@ -195,14 +195,47 @@ Char GET_NEXT_CHAR(void)
     }
     else
         STATE(In)++;
+
+    // handle line continuation, i.e., backslash followed by new line or CRLF;
+    // and also the end of a line in general
+    while (*STATE(In) == '\\' || *STATE(In) == 0) {
+        if (!*STATE(In))
+            GetLine();
+        else if (STATE(In)[1] == '\n') {
+            STATE(In) += 2;
```

Comment:
Maybe add a comment here which indicates that a line continuation was detected?

Also: the term "line continuation" should be documented somewhere (resp. be replaced by a different, documented name, if somebody has a good suggestion?)